authenticator/views.py

```diff
@@ -1,3 +1,19 @@
+#Django
 from django.shortcuts import render
+from django.contrib.auth import authenticate, login
+from django.contrib.auth.backends import BaseBackend
+
+
+def log_in(request):
+
+    username = request.POST['username']
+    password = request.POST['password']
+    user = authenticate(request, username=username, password=password)
+
+    if user is not none:
+        login(request, user)
+        return f'Succesful login'
+    else: 
+        return f'Unsuccesfrul login, please try again'
```

Comment:
Not an error, just a suggestion. When you write conditionals, try to mantain the flow of the code for better readibility. For example, instead of what you did:

```
if user is none
  return f`Unsuccessfull login, please try again`

login(request, user)
return f`Succesful login`
```
That way, the flow of the code is not broken and the readibility is better